index.js

```diff
@@ -279,7 +279,7 @@ function css (css, file) {
       if (file === 'stdin' && output) file = output
 
       // TODO: Unit test this
-      if (file !== 'stdin') options.from = file
+      options.from = file === 'stdin' ? path.join(process.cwd(), 'stdin.css') : file
```

Comment:
The issue at hand is that some plugins do `path.dirname(options.from)` to get the containing folder as the basis for resolving other files relative to the CSS. So yes, we need a dummy filename. Perhaps it should be `stdin` instead of `stdin.css` to avoid confusion (especially with SugarSS)?